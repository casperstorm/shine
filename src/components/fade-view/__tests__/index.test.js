import renderer from 'react-test-renderer'
import React from 'react'

import FadeView from '../index'

describe('fade-view', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<FadeView />)
    expect(tree).toMatchSnapshot()
  })
})
