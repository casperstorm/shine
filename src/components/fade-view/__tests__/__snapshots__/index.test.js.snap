// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fade-view should render correctly 1`] = `
<View
  collapsable={undefined}
  style={
    Object {
      "opacity": 0,
    }
  }
/>
`;
